testing-overhead/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+  id("java")
```

Comment:
Can we avoid the yet another gradle wrapper and use `otel.java-conventions`? Any particular reason for the separation?

Reply:
Would that imply adding this subdir as a submodule of the main project? 

My desire for keeping this stand-alone is primarily motivated by the massive size of the base project and the fact that it commonly takes 9000 years to open in IDEA and index.  It's *MUCH* easier/simpler/faster to work in this project when the rest of the instrumentation is not in scope.

I also understand/disgust in having so many gradle wrappers in the repo...but is it really worse than having everything in one MASSIVE gradle project? I dunno. Curious what others think.

Reply:
If adding one gradle dependency to this separate project requires re-importing of all our 100s of instrumentation - that is much larger overkill to me than yet another gradle wrapper.

Also, don't forget that you don't actually _need_ a separate wrapper for every independent project. `../gradlwe build` works just fine.